service/routes.py

```diff
@@ -195,6 +198,31 @@ def delete_customers(customer_id):
         customer.delete()
     return make_response("", status.HTTP_204_NO_CONTENT)
 
+######################################################################
+# LOCK AN EXISTING CUSTOMER
+######################################################################
+@app.route("/customers/<int:customer_id>/lock", methods=["PUT"])
+def lock_customers(customer_id):
+    """
+    Lock a Customer
+    """
+    app.logger.info("Request to lock Customer with id: %s", customer_id)
+    check_content_type("application/json")
+    request_data = request.get_json()
+    customer = Customer.find(customer_id)
+    if not customer:
+        raise NotFound("customer with id '{}' was not found.".format(customer_id))
+    customer.id = customer_id
+    customer.first_name = request_data["first_name"]
```

Comment:
I'm not sure if we need to reset the first_name, last_name, username, and password. I think it might be ideal for this endpoint to not accept a JSON body, as I think it should only change the locked variable of a customer